build/builders.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Using Docker Build
+description: About building with existing builders
+keywords: build, buildx, buildkit
+---
+
+
+## Docker Build Components
+
+Currently Engine holds two build components:
+* Legacy Builder - Old builder implementation. It's executed through the `docker build` command.
+* Buildkit - Active by default in new and latest installations of Docker. It's executed through the `docker buildx build` command.
+
+### Docker Build Default Configuration
+
+The backend you have active depends on your Docker's installation.
+If you have installed:
+
+* __Docker Desktop (all platforms)__: BuildKit is enabled by default; no additional configuration is required.
+* In Linux, if you have:
+  * Engine 18.09 and higher you have BuildKit enabled by default.
+  * Engine releases older than 18.09 only include the Legacy Builder component, so we highly recommend you upgrade your installation or alternatively install Docker Desktop.
```

Comment:
These are really old version, and EOL for four years. In general, we write the docs for the current version, and should avoid old versions (unless there a very strong need)

Reply:
This is not in the PR after the last commit. @crazy-max will remove the builders file so we can rework it in a new PR, if there's anything on there that we need re config for example.